scripts/mcstatus.coffee

```diff
@@ -0,0 +1,48 @@
+# Description:
+#   Report the status of a Minecraft server
+#   Uses http://api.syfaro.net/
```

Comment:
I didn't know this existed, I had assumed this was going to use the Query port directly. :sparkles: